vumi_http_retry/workers/api/worker.py

```diff
@@ -42,30 +43,35 @@ def setup(self):
     def teardown(self):
         self.redis.transport.loseConnection()
 
-    @classmethod
-    def respond(cls, req, data, code=http.OK):
-        req.responseHeaders.setRawHeaders(
-            'Content-Type', ['application/json'])
-
-        req.setResponseCode(code)
-        return json.dumps(data)
-
     @app.route('/health', methods=['GET'])
     def route_health(self, req):
-        return self.respond(req, {})
+        return response(req, {})
 
     @app.route('/requests/', methods=['POST'])
+    @json_body
+    @validate(
+        has_header('X-Owner-ID'),
+        body_schema({
+            'properties': {
```

Comment:
I'm okay with it staying right next to the route. My thought was that
having it separate helps as validation gets complicated (e.g. one can
re-use it in different places, test validation separately, split out pieces
into separate variables for clarity and re-use). Let's keep it here for now
and adjust later if it feels needed.